File: Plugins/Aspose_Barcode_Java_for_PHP/src/aspose/barcode/WorkingWithBarcode/AdvanceBarcodeFeatures/AustraliaPostBarcode.php
<?php

namespace Aspose\Barcode\WorkingWithBarcode\AdvanceBarcodeFeatures;

use com\aspose\barcode\Symbology as Symbology;
use com\aspose\barcode\BarCodeBuilder as BarCodeBuilder;
use com\aspose\barcode\AustraliaPostFormatControlCode as AustraliaPostFormatControlCode;

class AustraliaPostBarcode{

    public static function run($dataDir=null){

        # Instantiate barcode object
        $symbology = new Symbology();
        $builder = new BarCodeBuilder("1234567890", $symbology->AustraliaPost);

        # Set format control code to standard
        $australiaPostFormatControlCode=new AustraliaPostFormatControlCode();
        $builder->setAustraliaPostFormatControlCode($australiaPostFormatControlCode->Standard);

        # Save the image to your system and set its image format to Jpeg
        $builder->save($dataDir . "AusraliaPost-Barcode.jpg");

        # Display Status
        print "Generated Austrailia Post Barcode Successfully.".PHP_EOL;
    }

}

File: Plugins/Aspose.BarCode Java for Jython/asposebarcode/WorkingWithBarcode/AdvanceFeatures/AustraliaPostBarcode.py
from asposebarcode import Settings
from com.aspose.barcode import BarCodeBuilder
from com.aspose.barcode import Symbology
from com.aspose.barcode import AustraliaPostFormatControlCode

class AustraliaPostBarcode:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithBarcode/AdvanceBarcodeFeatures/AustraliaPostBarcode/'
        
        # Instantiate barcode object
        symbology = Symbology
        builder = BarCodeBuilder("1234567890", symbology.AustraliaPost)

        # Set format control code to standard
        australiaPostFormatControlCode=AustraliaPostFormatControlCode
        builder.setAustraliaPostFormatControlCode(australiaPostFormatControlCode.Standard)

        # Save the image to your system and set its image format to Jpeg
        builder.save(dataDir + "AusraliaPost-Barcode.jpg")

        # Display Status
        print "Generated Austrailia Post Barcode Successfully."


if __name__ == '__main__':        
    AustraliaPostBarcode()